driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/kill.bpf.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(kill_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct ringbuf_struct ringbuf;
+	if(!ringbuf__reserve_space(&ringbuf, KILL_E_SIZE))
+	{
+		return 0;
+	}
+
+	ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_KILL_E, KILL_E_SIZE);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: pid (type: PT_PID) */
+	pid_t pid = (s32)extract__syscall_argument(regs, 0);
+	ringbuf__store_s64(&ringbuf, (s64)pid);
+
+	/* Parameter 2: sig (type: PT_SIGTYPE) */
+	u8 sig = (u8)extract__syscall_argument(regs, 1);
```

Comment:
According to the man page, the sig argument is signed (`int`). But I guess, u8 is due to the scap event definition then?